### PR TITLE
Fixing typo

### PR DESCRIPTION
**Pull request by peterabrahamdev**
*A typo got fixed where the word "Flutter" was written as "Fluter".*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Review by goderbauer — APPROVED** [review]
~LGM~ 

Edit: LGTM

File: examples/texture/test/texture_test.dart [hunk 1]
```diff
@@ -11,6 +11,6 @@ void main() {
         .main(); // builds the app and schedules a frame but doesn't trigger one
     await tester.pump(); // triggers a frame
 
-    expect(find.text('Fluter Blue'), findsOneWidget);
```
Comment (goderbauer) [inline examples/texture/test/texture_test.dart:14]: Actually, why was this test not failing or is not failing now? Is it potentially not running on CI?

**Review by Renzo-Olivares — APPROVED** [review]
LGTM

File: examples/texture/test/texture_test.dart (hunk 1 above)
Comment (goderbauer) [reply examples/texture/test/texture_test.dart:14]: Potential fix to run all tests: https://github.com/flutter/flutter/pull/138374

**Comment by goderbauer** [comment]
Let's hold off with submitting this until it is verified that the text with the typo fails in https://github.com/flutter/flutter/pull/138374.

**Pull request merged**